src/http.ts

```diff
@@ -16,6 +16,7 @@ export interface RequestOpts {
   body?: any
   query?: Object
   headers?: StringMap
+  cache?: 'default' | 'no-store' | 'reload' | 'no-cache' | 'force-cache'
```

Comment:
You can use `RequestCache` interface instead, which comes with the standard lib.